MapleServer2/Types/Player.cs

```diff
@@ -109,12 +106,11 @@ public Player()
         public static Player Char1(long accountId, long characterId, string name = "Char1")
         {
             int job = 50; // Archer
-
             PlayerStats stats = PlayerStats.Default();
             StatDistribution StatPointDistribution = new StatDistribution();
             List<SkillTab> skillTabs = new List<SkillTab>
             {
-                XmlParser.ParseSkills(job)
+                new SkillTab((Job) (job*10))
```

Comment:
should we make constants for this?

Reply:
Sure! changing on player the `int Job` to the `Job job` and using it on the Packet for character too